packages/remix/src/build.ts

```diff
@@ -428,14 +428,20 @@ module.exports = config;`;
       : null,
   ]);
 
-  const staticDir = join(
-    remixConfig.assetsBuildDirectory,
-    ...remixConfig.publicPath
-      .replace(/^\/|\/$/g, '')
-      .split('/')
-      // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      .map(_ => '..')
-  );
+  const assetsBuildDirectory = remixConfig.assetsBuildDirectory
+    .replace(/^\/|\/$/g, '')
+    .split('/');
+  const publicPath = remixConfig.publicPath.replace(/^\/|\/$/g, '').split('/');
```

Comment:
I tested this last week and if there's no `publicPath`, the path does not start with a `/` (or `C:\`). This leads me to wonder if there's a better way to use `path.relative()` and `path.join()`. I'll be in touch.